### DISCUSSION TITLE
How to properly configure fonts?

### DISCUSSION THREAD

**Discussion by zer09** [wezterm/wezterm#2709] [Q&A] [answered]
I a using this font https://github.com/sainnhe/icursive-nerd-font/tree/master/dist/Lexsa
But when I configured it will only use the italic version after I restart my PC.

I have this configuration.
```lua
	-- Font configuration
	font = wezterm.font('Lexsa'),
	font_size = 11.0,
	line_height = 0.8,
```
But the output is this, its all italic
![image](https://user-images.githubusercontent.com/780780/200148342-32189f3e-d779-4ac8-967d-c582dab7b456.png)

System Details:
wezterm 20220905-102802-7d4b8249
Windows 11	10.0.22621 Build 22621

**Comment by wez** [comment, marked as answer]
Make sure that you've installed all of the different font variants for that family (not just the italic version).
Take a look at https://wezfurlong.org/wezterm/config/fonts.html#troubleshooting-fonts

**Reply by zer09** [reply]
Thanks got it working, but I need to install the font to all users before wezterm able to find it. The font didnt show in list-fonts --list-system until I install it for all users
